src/app/machines/views/MachineList/MachineListTable/NameColumn/NameColumn.test.tsx

```diff
@@ -145,57 +97,30 @@ describe("NameColumn", () => {
       { ip: "127.0.0.1", is_boot: false },
       { ip: "127.0.0.1", is_boot: false },
     ];
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter
-          initialEntries={[{ pathname: "/machines", key: "testKey" }]}
-        >
-          <CompatRouter>
-            <NameColumn groupValue={null} systemId="abc123" />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
-    );
-    expect(wrapper.find('[data-testid="ip-addresses"]').text()).toBe(
-      "127.0.0.1"
+    renderWithBrowserRouter(
+      <NameColumn groupValue={null} systemId="abc123" />,
+      { route: "/machines", state }
     );
-    // Doesn't show toolip.
-    expect(wrapper.find("Tooltip").exists()).toBe(false);
+    expect(screen.getByTestId("ip-addresses")).toHaveTextContent("127.0.0.1");
+    expect(screen.queryByTestId("Tooltip")).not.toBeInTheDocument();
   });
 
   it("can show a single mac address", () => {
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter
-          initialEntries={[{ pathname: "/machines", key: "testKey" }]}
-        >
-          <CompatRouter>
-            <NameColumn groupValue={null} showMAC={true} systemId="abc123" />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
+    renderWithBrowserRouter(
+      <NameColumn groupValue={null} showMAC={true} systemId="abc123" />,
+      { route: "/machines", state }
     );
-    expect(wrapper.find("a").text()).toEqual("00:11:22:33:44:55");
+    expect(screen.getByText(/00:11:22:33:44:55/i)).toBeInTheDocument();
```

Comment:
This was previously checking if this is a link.
```suggestion
    expect(screen.getByRole("link", { name: /00:11:22:33:44:55/i })).toBeInTheDocument();
```